vscode-car-plugin/src/main/java/org/wso2/maven/CAppHandler.java

```diff
@@ -121,8 +121,15 @@ void processArtifactsInFolder(File artifactsDir, List<ArtifactDependency> depend
                         if (Constants.API_TYPE.equals(type)) {
                             // api version can be null
                             apiList.put(name, configVersion);
-                            writeMetadataFile(name, configElement, archiveDirectory, false, version);
-                            addMetadataDependencies(metadataDependencies, configElement, false, version);
+                            String metadataFilename = name + "_metadata.yaml";
```

Comment:
Did we check with versioned APIs?

Reply:
Fixed in 3572327181bcdf2a03c02ccbe0c0e2355271e1f8